static_src/components/service_instance_list_panel.jsx

```diff
@@ -0,0 +1,54 @@
+
+import style from 'cloudgov-style';
+import React from 'react';
+
+import PanelRow from './panel_row.jsx';
+import ServiceInstance from './service_instance.jsx';
+
+import createStyler from '../util/create_styler';
+
+const propTypes = {
+  serviceInstances: React.PropTypes.array,
```

Comment:
So this doesn't end up in app container (which I'd agree shouldn't take care of everything), it ends up in `service_instance_panel`. The reason I did it like this was so each instance list could get a bunch of instances and  wouldn't worry if they're bound or not. But I ended up not adding "bound" as a property on the instance so I still have to pass whether they're bound or not. 

One benefit of it being here rather then each instance list; I think they'd both have to do the fetching of the same elements, where here it's just done once.